operations/validate_csm_health.md

```diff
@@ -55,6 +55,29 @@ If running these checks after the initial CSM install, then to find details on c
 
 All platform health checks are expected to pass. Each check has been implemented as a [Goss](https://github.com/aelsabbahy/goss) test which reports a `PASS` or `FAIL`.
 
+If the CSM version is 1.6.0 or lower, then before running health checks, restart the `goss-servers` service on all NCNs to ensure the correct tests are run on each node. This is necessary because of a timing issue that is fixed in CSM 1.6.1.
+
+> Skip the `goss-servers` service restart if the CSM version is 1.6.1 or above.
+> The restart will cause no harm if done on CSM 1.6.1 or higher, but it is unnecessary.
+Run one of the two commands below depending on if it is being executed from `ncn-m001` or from the PIT node.
```

Comment:
We need this whitespace here, or else the "Run..." part will be part of the previous comment, which we do not want.

```suggestion

Run one of the two commands below depending on if it is being executed from `ncn-m001` or from the PIT node.
```